dbms/src/IO/ReadBufferFromS3.cpp

```diff
@@ -1,65 +1,40 @@
 #include <IO/ReadBufferFromS3.h>
 
 #include <IO/ReadBufferFromIStream.h>
-#include <IO/S3Common.h>
 
 #include <common/logger_useful.h>
+#include <aws/s3/model/GetObjectRequest.h>
```

Comment:
I think we need to add `#ifdef` here, because ClickHouse must be able to build without S3 support.

Reply:
Ok, will do it as with `StorageHDFS`.